ROADMAP.md

```diff
@@ -0,0 +1,47 @@
+# Sealer roadmap
+
+## v0.1.3
+
+* general e2e tests
+* support build, apply and push a CloudImage
+
+## v0.2.0
+
+* support cluster ENV and global config
+* support ARM base image
+* registry cache docker image when pull images
+* base image support more kubernetes versions and auto make rootfs
```

Comment:
auto make rootfs? where to support this?

Reply:
this almost done by @cuisongliu , same as sealos auto make offline package.